### DISCUSSION TITLE
Solution to run a repository

### DISCUSSION THREAD

**Discussion by Edece88** [community/community#40644] [New to GitHub]
Would you be so kind to help me, I want instructions to install this repository on my pc https://github.com/Mika-IO/TRRB The instructions come but it doesn't tell me in which program if in visual code or command prompt to install it.

**Comment by queenofcorgis** [comment]
If you're asking how to install the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
